src/what-unsafe-does.md

```diff
@@ -16,18 +16,41 @@ to your program. You definitely *should not* invoke Undefined Behavior.
 Unlike C, Undefined Behavior is pretty limited in scope in Rust. All the core
 language cares about is preventing the following things:
 
-* Dereferencing null, dangling, or unaligned pointers
-* Reading [uninitialized memory][]
+* Dereferencing (using the `*` operator on) dangling, or unaligned pointers, or
+  wide pointers with invalid metadata (see below)
```

Comment:
So this is just editorial, right? LGTM. I'd prefer if we explained what "dereferencing" means. Specifically, I want to be sure that people consider `&*x` as dereferencing `x`.

> It would be nice if we could use "invalid" here to establish a blanket term for these properties,

Yeah a term would be nice. The problem is that "valid (raw) pointer" would be interpreted as referring to the validity invariant of a raw pointer (defined a bit further down), and that is not the same thing.

This *is*, however, the validity invariant of a reference that we are repeating here. So maybe we could say something like this?

"Dereferencing (using the `*` operator on) a pointer that is not a valid reference (see below)"